### PR TITLE
Ocr accessibility fixes

### PR DESCRIPTION
**Pull request by cbryant24**

### CODE REVIEW

File: app/javascript/src/navigation/uninav.js [hunk 1]
```diff
@@ -31,25 +31,27 @@ const uninav = function() {
...
-          $(this).attr("aria-expanded", "true");
+          $(this).find('.uninav__dropdown--child').attr("aria-expanded", "true");
         }
       }
     });
   }
 
   function hideCurrentDropdownWhenLoseFocus() {
     $(".uninav__dropdown--child li:last-of-type").on("keydown blur", function (e) {
+      debugger
```
Comment (nicholasnadel) [inline app/javascript/src/navigation/uninav.js:47]: @cbryant24 heads up

File: app/assets/javascripts/cascade/level/accordion.js [hunk 2]
```diff
@@ -21,9 +21,30 @@ $(function () {
...
+      return;
+    }
+
+    if ($('.accordion.active').length === 0) {
+      $('.accordion').attr('aria-expanded', 'false');
+      $('.toggle-expand-collapse').removeClass('collapse').addClass('expand');
+      $('.toggle-expand-collapse').text('Expand');
+      return;
+    }
+    debugger
```
Comment (nicholasnadel) [inline app/assets/javascripts/cascade/level/accordion.js:40]: @cbryant24 another

File: app/assets/javascripts/cascade/level/accordion.js [hunk 3]
```diff
@@ -32,6 +53,7 @@ $(function () {
   })
   // HANDLE EXPAND TOGGLES
   $(".collapsibles-widget .toggle-expand-collapse").on('click keypress', function (event) {
+    debugger
```
Comment (nicholasnadel) [inline app/assets/javascripts/cascade/level/accordion.js:56]: @cbryant24

File: app/javascript/src/navigation/uninav.js (hunk 1 above)
Comment (cbryant24) [reply app/javascript/src/navigation/uninav.js]: @nicholasnadel should be removed, good catch

File: app/assets/javascripts/cascade/level/accordion.js (hunk 2 above)
Comment (cbryant24) [reply app/assets/javascripts/cascade/level/accordion.js]: @nicholasnadel should be removed, good catch

File: app/assets/javascripts/cascade/level/accordion.js (hunk 3 above)
Comment (cbryant24) [reply app/assets/javascripts/cascade/level/accordion.js]: @nicholasnadel should be removed, good catch

**Review by nicholasnadel — APPROVED** [review]

**Review by corejh — APPROVED** [review]

**Pull request merged**